x/protorev/client/cli/tx.go

```diff
@@ -141,22 +141,34 @@ func CmdSetMaxPoolPointsPerBlock() (*osmocli.TxCliDesc, *types.MsgSetMaxPoolPoin
 	}, &types.MsgSetMaxPoolPointsPerBlock{}
 }
 
-// CmdSetPoolWeights implements the command to set the pool weights used to estimate execution costs
-func CmdSetPoolWeights() *osmocli.TxCliDesc {
+// CmdSetInfoByPoolType implements the command to set the pool information used throughout the module
+func CmdSetInfoByPoolType() *osmocli.TxCliDesc {
 	desc := osmocli.TxCliDesc{
-		Use:   "set-pool-weights [path/to/routes.json]",
-		Short: "set the protorev pool weights",
-		Long: `Must provide a json file with all the pool weights that will be set. 
+		Use:   "set-info-by-pool-type [path/to/routes.json]",
+		Short: "set the protorev pool type info",
+		Long: `Must provide a json file with all the pool info that will be set. 
 		Sample json file:
 		{
-			"stable_weight" : 1,
-			"balancer_weight" : 1,
-			"concentrated_weight" : 1
+			"stable" : {
+				"weight" : 1,
+			},
+			"concentrated" : {
+				"weight" : 1,
+				"max_ticks_crossed": 10,
+			},
+			"balancer" : {
+				"weight" : 1,
+			},
+			"cosmwasm" : {
+				"weight_map" : {
+					"ibc/123..." : 1,
```

Comment:
```suggestion
					"osmo123..." : 1,
```

Reply:
nit: cosmwasm weight map represents address to weight, so use an address placeholder instead of an asset placeholder